the-name-card/src/components/WidgetItem/Items/Link.tsx

```diff
@@ -0,0 +1,53 @@
+import { z } from 'zod';
+
+import { WidgetProp } from '.';
+import './Link.scss';
+
+const LinkValues = z.object({ description: z.string(), link: z.string() });
+
+export type LinkValues = z.infer<typeof LinkValues>;
+
+export const Link: React.FC<WidgetProp> = ({ value, updateValue }) => {
+	const rawValue = JSON.parse(value);
+	const linkValues = LinkValues.parse(rawValue);
+
+	const editDescription = (
+		type: 'description' | 'link',
```

Comment:
nit: possible to abstract out these types somewhere

Reply:
will do